modules/Sources/Dependencies/AddressBookClient/AddressBookLiveKey.swift

```diff
@@ -39,18 +39,19 @@ extension AddressBookClient: DependencyKey {
                 }
                 
                 // contacts haven't been loaded from the remote storage yet, do it
-                do {
-                    let data = try await remoteStorage.loadAddressBookContacts()
-
-                    let storedContacts = try AddressBookClient.decryptData(data)
-                    latestKnownContacts.value = storedContacts
-
-                    return storedContacts
-                } catch RemoteStorageClient.RemoteStorageError.fileDoesntExist {
-                    return []
-                } catch {
-                    throw error
-                }
+//                do {
+//                    let data = try await remoteStorage.loadAddressBookContacts()
+//
+//                    let storedContacts = try AddressBookClient.decryptData(data)
+//                    latestKnownContacts.value = storedContacts
+//
+//                    return storedContacts
+//                } catch RemoteStorageClient.RemoteStorageError.fileDoesntExist {
+//                    return []
+//                } catch {
+//                    throw error
+//                }
```

Comment:
The same comment here. Should we remove this, or do we have a plan with the commented-out code?

Reply:
All commented code will be removed in a followup ticket (once other PRs are merged to the feature branch).